src/style.css

```diff
@@ -0,0 +1,93 @@
+@font-face {
+  font-family: "Avenir-Book";
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+  font-style: normal;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Avenir-Book", "Avenir-Heavy", sans-serif;
```

Comment:
You need 2 different font families here. if first doesn't work (second doesn't work to because it's one family "Avenir" and will work only default san-serif)